### PR TITLE
[keymgr/dv] Update checklist and move to V1

### PR DESCRIPTION
**Pull request by weicaiyang**
Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

File: hw/ip/keymgr/doc/checklist.md [hunk 1]
```diff
@@ -115,28 +115,28 @@ Review        | Signoff date            | Not Started |
...
 Code Quality  | [TB_LINT_SETUP][]                     | Done        |
-Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | Not Started |
-Review        | [DESIGN_SPEC_REVIEWED][]              | Not Started |
-Review        | [DV_PLAN_TESTPLAN_REVIEWED][]         | Not Started |
-Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Not Started | Exception (?)
-Review        | [V2_CHECKLIST_SCOPED][]               | Not Started |
+Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | N/A         |
+Review        | [DESIGN_SPEC_REVIEWED][]              | Done        |
+Review        | [DV_PLAN_TESTPLAN_REVIEWED][]         | Done        |
+Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Done        | Exception (Security, Power, Performance, Debug)
```
Comment (sriyerg) [inline hw/ip/keymgr/doc/checklist.md:138]: No "security" features in keymgr?

File: hw/ip/keymgr/doc/checklist.md [hunk 2]
```diff
@@ -115,28 +115,28 @@ Review        | Signoff date            | Not Started |
...
+Testbench     | [TB_TOP_CREATED][]                    | Done        |
+Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Done        |
+Testbench     | [SIM_TB_ENV_CREATED][]                | Done        |
+Testbench     | [SIM_RAL_MODEL_GEN_AUTOMATED][]       | Done        |
+Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Done        |
+Testbench     | [TB_GEN_AUTOMATED][]                  | N/A         |
+Tests         | [SIM_SMOKE_TEST_PASSING][]            | Done        |
+Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | Done        |
+Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | N/A         |
+Tool Setup    | [SIM_ALT_TOOL_SETUP][]                | Done        |
```
Comment (sriyerg) [inline hw/ip/keymgr/doc/checklist.md:129]: I assume the smoke test passing with Xcelium?

File: hw/ip/keymgr/doc/checklist.md (hunk 1 above)
Comment (weicaiyang) [reply hw/ip/keymgr/doc/checklist.md:138]: security isn't fully planned. Will we use FPV or just DV for the security, which is not very clear to me now. Let me change the description here.

File: hw/ip/keymgr/doc/checklist.md (hunk 2 above)
Comment (weicaiyang) [reply hw/ip/keymgr/doc/checklist.md:129]: yes, I have also cleaned up some xcelium warnings too.

**Review by sriyerg — APPROVED** [review]
LGTM, pending one change.

**Pull request merged**